src/PerformanceTests/Common/Scenarios/SendLocalOneOnOne/SendLocalOneOnOneRunner.cs

```diff
@@ -16,9 +16,7 @@ partial class SendLocalOneOnOneRunner : BaseRunner
 
     protected override void Start()
```

Comment:
You are right about the async/sync workflow. That is something to consider. @WilliamBZA @dvdstelt Your opinion?